docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -65,11 +65,11 @@ You can also download and deploy ELSER to an {infer} pipeline directly from the
 2. Select the index from the list that has an {infer} pipeline in which you want 
 to use ELSER.
 3. Navigate to the **Pipelines** tab.
-4. Under **{ml-app} {infer-cap} Pipelines**, click the "Start single-threaded" 
-button to download and start the model with basic configuration or select the 
-"Fine-tune performance" option to navigate to the **Trained Models** page where 
-you can configure the model deployment.
-
+4. Under **{ml-app} {infer-cap} Pipelines**, click "Deploy" button to begin downloading
```

Comment:
Since there are two steps for Enterprise Search as well as the other methods it probably makes sense to structure this whole docs page similarly for all 3 methods.

At the moment we have high level "Download" and "Deploy" sections with the two steps split for the ML UI and dev console but Enterprise Search's two steps all in the first section. So it's probably better to either just have one high level section and put all the steps for each method in one list for that method. Or else split Enterprise Search too.

This also highlights that there is different terminology in different apps. ML is using "deploy" to mean "start" whereas Enterprise Search is using "deploy" to mean "download".

@szabosteve maybe you should take over this PR to do a broader refactoring.

Reply:
Ack. Will do today.

Reply:
Hey @szabosteve, it's all good 🙂  Yep, I'll forward-port.